examples/methods/methods.go

```diff
@@ -34,4 +43,12 @@ func main() {
 	rp := &r
 	fmt.Println("area: ", rp.area())
 	fmt.Println("perim:", rp.perim())
+
+	// Consider what may happen if `setLabel` worked on a value
```

Comment:
I don't know if "Consider what may happen ..." is really useful for learners here. I'd jus say something shorter about setLabel being able to modify `r` because it takes a reference to the actual object rather than a copy (as with value receivers).

Reply:
Good point.

Also, I added the comment about pointer receiver methods being more efficient because I read that somewhere when doing 5 minutes of research after doing this exercise.  I wanted to give the reader a "if you don't know what to use or it doesn't matter, just use *pointer* receivers" fact to latch on to.  Is this generally true?